server/conference/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class ConferenceNotFound(Exception):
+    pass
+
+
+class ConferenceControllerError(Exception):
+    pass
+
+
+class ForbiddenConferenceAction(Exception):
```

Comment:
Bad naming. This exception's naming doesn't represent its exception nature. This sounds more like an `Action` child class. Should be renamed to `ForbiddenConferenceActionError`

Reply:
Fixed